.github/workflows/builder_maven_slsa3.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
define the permissions `permissions: {}`

Reply:
feel free to add it to the gradle workflow

Reply:
feel free to add it to the gradle workflow and close https://github.com/slsa-framework/slsa-github-generator/issues/2177